docs/scenarios.md

```diff
@@ -474,30 +483,31 @@ which contains a green-screen intended for adversarial patch insertion. The data
-|:----------:|:-----------:|:---------------------------:|:--------------------------------:|:-------------------------------:|:---------------------------:|:----------------:|:--------------------------------:|:------------------------------------:|:-----------------------------------:|:-------------------------------:|:---------:|
-|    Small   |  0.43/0.40  |          0.37/0.45          |              0.8/1.0             |            0.06/0.08            |          0.57/0.47          |     0.19/0.21    |             0.40/0.48            |                7.6/7.6               |              0.06/0.08              |            0.54/0.45            |     10    |
-|   Medium   |  0.48/0.37  |          0.39/0.50          |              1.2/1.3             |            0.01/0.01            |          0.60/0.49          |     0.24/0.16    |             0.39/0.51            |               12.4/6.8               |              0.01/0.01              |            0.61/0.48            |     10    |
-|    Large   |  0.38/0.31  |          0.36/0.43          |              1.0/1.0             |            0.05/0.02            |          0.59/0.55          |     0.17/0.14    |             0.37/0.34            |               10.3/10.2              |              0.05/0.02              |            0.57/0.55            |     10    |
+| Attack            | Benign  mAP | Benign  Disappearance  Rate | Benign  Hallucination  per Image | Benign  Misclassification  Rate | Benign  True Positive  Rate | Adversarial  mAP | Adversarial  Disappearance  Rate | Adversarial Hallucination  per Image | Adversarial Misclassification  Rate | Adversarial True Positive  Rate | Test Size |
+|-------------------|-------------|-----------------------------|----------------------------------|---------------------------------|-----------------------------|------------------|----------------------------------|--------------------------------------|-------------------------------------|---------------------------------|-----------|
+| Robust DPatch     | 0.76/0.72   | 0.19/0.22                   | 3.97/3.48                        | 0.06/0.06                       | 0.75/0.71                   | 0.68/0.66        | 0.27/0.28                        | 4.48/3.65                            | 0.06/0.07                           | 0.67/0.65                       | 31        |
+| Adversarial Patch | 0.76/0.72   | 0.19/0.22                   | 3.97/3.48                        | 0.06/0.06                       | 0.75/0.71                   | 0.54/*           | 0.32/*                           | 22.16/*                              | 0.05/*                              | 0.62/*                          | 31        |
```

Comment:
For `carla_obj_det_adversarialpatch_undefended.json`, I'm seeing:

```
"adversarial_carla_od_AP_per_class": [
            {
                "class": {
                    "1": 0.43,
                    "2": 0.41,
                    "3": 0.6
                },
                "mean": 0.48
            }
        ],
```

But the doc lists `0.54` for `Adversarial mAP`. This is a less drastic discrepancy, but perhaps nontrivial. The more granular metrics align closely with my results, as do the benign results

Reply:
@lcadalzo These are the `learning_rate` I used to obtain my results, for both single and multimodality, defended and undefended.

`Adversarial Patch`: `learning_rate = 0.003`
`Robust DPatch`: `learning_rate = 0.002`